app-frontend/src/app/components/pages/project/analyses/visualize/index.html

```diff
@@ -2,9 +2,12 @@
     <div class="page-header wide">
         <h3>Key metrics</h3>
     </div>
-    <div class="page-card analysis-maps-card">
+    <div class="page-card analysis-histogram-card text-center" ng-if="$ctrl.isLoadingAnalysis">
+      <h3>Loading histograms and maps<i class="icon-load animate-spin"></i></h3>
+    </div>
```

Comment:
We should add an error case for when histograms fail to fetch. This will definitely happen at some point, either due to a bad analysis, tile server instability etc.